faiss/gpu/test/TestGpuIndexCagra.cu

```diff
@@ -198,7 +198,7 @@ void copyToTest(
         faiss::MetricType metric,
         double expected_recall,
         bool base_level_only) {
-    for (int tries = 0; tries < 5; ++tries) {
+    for (int tries = 0; tries < 100; ++tries) {
```

Comment:
Do we still need to increase this or is this a testing artifact? I think this increases probability but does not make this certain.

Reply:
Testing artifact, just to see if we get intermittent accuracy drops. I'll change this once we are comfortable merging this PR in.